app/controllers/PostProcess.scala

```diff
@@ -98,7 +98,7 @@ object PostProcess extends JSONParser {
     toSimpleVertexArrJson(queryResultLs, Seq.empty[QueryResult])
   }
 
-  def toSimpleVertexArrJson(queryResultLs: Seq[QueryResult], exclude: Seq[QueryResult]): JsValue = {
+  def toSimpleVertexArrJson(queryResultLs: Seq[QueryResult], exclude: Seq[QueryResult], limit: Int = Query.defaultLimit): JsValue = {
```

Comment:
@elric-k 
I know it is weired, but currently, QueryResult has Query as member variable. so once Query instance has list parameter in it, personally, it is better to get global limit from query instance, rather than explicitly get parameter. what do you think?

May be we can create new issue to refactor QueryResult.